File: src/main/java/com/home/demos/samplebankmodule/repositories/cards/CardRepository.java
package com.home.demos.samplebankmodule.repositories.cards;

import com.home.demos.samplebankmodule.repositories.cards.entities.Card;
import com.home.demos.samplebankmodule.repositories.payments.entities.Payment;

import java.util.List;

public interface CardRepository {
    List<Card> findAllByClientId(Long clientId);

    Card save(Card card);
}

File: src/main/kotlin/com/home/demos/samplebankmodule/repositories/cards/CardRepository.kt
package com.home.demos.samplebankmodule.repositories.cards

import com.home.demos.samplebankmodule.model.Card
import com.home.demos.samplebankmodule.repositories.Repository

interface CardRepository : Repository {
    fun findAllByClientId(clientId: Long): List<Card>
    fun save(card: Card): Card
}
